cpp/include/cugraph/detail/graph_utils.cuh

```diff
@@ -47,6 +48,22 @@ struct compute_gpu_id_from_vertex_t {
   }
 };
 
+template <typename vertex_t>
+struct compute_gpu_id_from_int_vertex_t {
+  vertex_t const* vertex_partition_range_lasts;
+  size_t num_vertex_partitions;
```

Comment:
yeah... maybe just a FIXME statement, but we should eventually replace this (pointer, size) pairs to raft::device_span.

Reply:
Changed to span in the next push.